Keas.Mvc/ClientApp/components/Equipment/EquipmentEditValues.tsx

```diff
@@ -17,7 +17,8 @@ interface IProps {
 }
 
 export default class EquipmentEditValues extends React.Component<IProps, {}> {
-    public render() {
+    public render() {
+        const typeValue = this.props.selectedEquipment.type ? this.props.selectedEquipment.type : "Default";
```

Comment:
@jSylvestre usually we use this syntax `const typeValue = this.props.selectedEquipment.type || "Default";`

Reply:
@srkirkland Changed. But just monkey see monkey do:
https://github.com/ucdavis/Keas/blob/master/Keas.Mvc/ClientApp/components/Keys/KeySerialEditValues.tsx#L25